forc/src/ops/forc_init.rs

```diff
@@ -78,7 +78,7 @@ pub fn init(command: InitCommand) -> ForcResult<()> {
             .into_owned(),
     };
 
-    validate_name(&project_name, "project name")?;
```

Comment:
We should also update this in `new.rs`, `forc_template.rs`, and `manifest/mod.rs:553`